kolibri/plugins/learn/assets/src/vue/breadcrumbs/breadcrumb/index.vue

```diff
@@ -1,11 +1,9 @@
 <template>
 
   <a v-link="linkobject">
-    <span class='back'>
-      <span v-if="showarrow" class="visuallyhidden">{{ $tr('back') }}</span>
-      <svg v-if="showarrow" role="presentation" src="../../icons/back.svg"></svg>
-      {{ text }}
-    </span>
+    <span v-if="showarrow" class="visuallyhidden">{{ $tr('back') }}</span>
```

Comment:
I would also suggest to give a bit more context to the user here, since there's no concern for the length of the "invisible" string, so maybe `Back to previous content group` or similar instead of only `back`. :wink: